spinner/spinner.go

```diff
@@ -0,0 +1,13 @@
+package spinner
+
+// This is a wrapper around spinner to support different operation systems until upstream is fixed.
```

Comment:
What upstream are we talking about? Is this comment relevant? If so, could you please link to a relevant open issue?